pkg/logger/logger.go

```diff
@@ -51,6 +51,10 @@ func New() (Logger, error) { return defaultConfig.New() }
 func (c *Config) New() (Logger, error) {
 	cfg := zap.NewProductionConfig()
 	cfg.Level.SetLevel(c.Level)
+	return NewFromZap(cfg)
+}
+
+func NewFromZap(cfg zap.Config) (Logger, error) {
```

Comment:
Another option is a func to modify the config:
```suggestion
func NewFromZap(cfgFn func(*zap.Config)) (Logger, error) {
```
This way the default can continue to evolve and things will stay in sync without the caller updating on their end... :thinking:

Reply:
Done